### DISCUSSION TITLE
Migrating vorta deb to vorta flatpak

### DISCUSSION THREAD

**Discussion by HDInfautre** [borgbase/vorta#1691] [FAQ]
hello
I had a lot of problems getting Vorta /flatpak to work under ubuntu 22.10.
I managed to get Vorta v0.8.7 .deb working
1/ I want to know how to migrate from one version to another version in another format?
Today v.0.87/deb => v.0.8.12/flatpak. How to do this, tutorial ?

2/ will there be updated versions for .deb?

3/ see appimage (more portable for me) ?
regards

**Comment by real-yfprojects** [comment]
As long as the data directory is the same vorta does the migrations itself.

**Reply by HDInfautre** [reply]
Hi 
So I did the vorta 0.8.12/flatpak installation.
The installation went well
Good news: I can now create the SSH key directly.
Bad news : I didn't get anything from the Vorta /deb version (I can't launch this version anymore)

Anyway, I've been in test mode for weeks!
I would like to know if I can store my ssh keys under my old ubuntu 22.04 version without any problem, in order to avoid my repeated problems (and especially in the problems under my old version):
1-a) my ssh keys under :
/media/rv/donnees/Com. informat./1, My data/Export from computer/SSH keys/?
1-b) the Vorta configuration (to avoid import-export):
/media/rv/donnees/Com. informat./1, My data/config Vorta/

Can you confirm me if my 2 ideas are possible? which would allow me to be more secure in case of problem.
I don't have any backup in the clouds since months (considering my difficulties with the old version), which can be serious.

**Reply by real-yfprojects** [reply]
> Can you confirm me if my 2 ideas are possible? 

I don't fully get what you are trying to do. However it should be possible to use you old Vorta database (`settings.db`) with any equal or newer version. Vorta simply leverages the ssh configuration of the host system. So just search on the internet how to setup ssh keys on Ubuntu and you are good to go.

**Comment by HDInfautre** [comment]
Hi
Thank you for your reply.
Where is this data directory? Because I would have to backup it.
Or better how can I move it to my data drive?
regards

**Reply by real-yfprojects** [reply]
They should be located at `/home/user/.local/share/Vorta`.
